test/integration/forms/form.js

```diff
@@ -7,15 +7,21 @@ context('Patient Form', function() {
       .get('[data-nav-region]')
       .should('not.be.visible');
 
-    // TODO: replace with on-screen navigation when available
     cy
-      .get('[data-nav-region]')
-      .find('[data-patients-region] .app-nav__link')
-      .last()
-      .click({ force: true });
+      .wait(2000);
 
     cy
-      .get('[data-nav-region]')
-      .should('be.visible');
+      .get('iframe')
+      .should($iframe => {
```

Comment:
`its('src').should('contain'` I think

Reply:
gave that a shot, but was getting a 'could not find property' of src

Reply:
hmm maybe its doesn't work on attributes.. how about `should('have.attr'`?